postprocess.mjs

```diff
@@ -0,0 +1,100 @@
+// @ts-check
+import * as esbuild from "esbuild";
+import * as UglifyJS from "uglify-js";
+
+/**
+ * @type {import("elm-watch/elm-watch-node").Postprocess}
+ */
+export default async function postprocess({ code, targetName, compilationMode }) {
+  switch (compilationMode) {
+    case "standard":
+    case "debug":
+      return code;
+
+    case "optimize":
+      return minify(code, {
+        minimal: !["Html", "Sandbox", "Element", "Document", "Worker"].includes(targetName),
```

Comment:
This needs to be adjusted for the target names in your elm-watch.json. The thing here is that `minimal: true` gives smaller output but takes much longer. For a customer facing app that can be worth it, but for an internal app size might not matter as much and it’s more worth to have a speedy setup.